guides/source/command_line.md

```diff
@@ -395,6 +403,12 @@ With the `helper` method it is possible to access Rails and your application's h
 
 INFO: You can also use the alias "db" to invoke the dbconsole: `bin/rails db`.
 
+If you are using multiple database with Active Record, you can connect to command-line client by specifying the database name using `--database` or `--db`. Rails by defaults connects to the primary database.
```

Comment:
Since the preceding paragraph talks about how smart `bin/rails dbconsole` is, I thought it might make sense to mention the default behavior first instead:

```suggestion
If you are using multiple databases, `bin/rails dbconsole` will connect to the primary database by default.  You can specify which database to connect to using `--database` or `--db`:
```

What do you think?

Reply:
This sounds way better than what I had written. I'll make the change. 😊

Reply:
Done.